src/main/java/Duke/Command/MarkCommand.java

```diff
@@ -0,0 +1,43 @@
+package Duke.Command;
+
+import Duke.Command.Command;
+import Duke.Exceptions.InvalidArgumentException;
+import Duke.Storage;
+import Duke.TaskList;
+import Duke.Ui;
+
+public class MarkCommand extends Command {
+
+    private int mark;
```

Comment:
I believe int mark here refers to whether the command is marking a task as done (`mark == 1`) or unmarking/marking as not done (`mark == 0`). In this case, since `mark` can only take two possible values, I think it is better if `mark` is a boolean variable. You could also add javadocs here explaining what `mark` means to avoid confusion.